docs/introduction/dev-environment-quick-start-guide.md

```diff
@@ -184,3 +184,25 @@ You need to remove current directory where you clone the project, and clone the
 
 ```console
 git clone -c core.symlinks=true git@github.com:openfoodfacts/openfoodfacts-server.git
+
+```console
+git clone -c core.symlinks=true git@github.com:openfoodfacts/openfoodfacts-server.git
+
+
+### make dev error: [create_mongodb_indexes] Error 126 - C:/Program Files/Git/usr/bin/sh: no such file or directory: unknown
+
+When running "make dev":
+
+```console
+ðŸ¥« Creating MongoDB indexes â€¦
+docker cp conf/mongodb/create_indexes.js 1aabb19cd6ad30ecf50ef24b7bc5f5e9fe60e69b218ea0db8273f4e8ec01a1aa:/data/db
+docker-compose --env-file=.env exec -T mongodb /bin/sh -c "mongo off /data/db/create_indexes.js"
+OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "C:/Program Files/Git/usr/bin/sh": stat C:/Program Files/Git/usr/bin/sh: no such file or directory: unknown
+make: *** [create_mongodb_indexes] Error 126
+```
+
+Solution:
+Go to openfoodfacts-server and open the Makefile.
+Search for '/bin/sh', and replace it with '//bin/sh' - double the slashes. (There will be three /bin/sh, replace all three) 
```

Comment:
I suggested to Pradumna the heading // after looking for solutions to the error message, I stumbled upon this: https://stackoverflow.com/questions/68795453/git-sh-exe-docker-run-no-such-file-or-directory

So I'm not sure if the single quotes work as well, it would have to be tested.

Reply:
Alright, I will test it out. Then probably documenting this will help. Can you merge this dev-environment-quick-start-guide.md. 
After that, if testing works out, we can wan merge the Makefile as well and close the issue?

Reply:
> I suggested to Pradumna the heading // after looking for solutions to the error message, I stumbled upon this: https://stackoverflow.com/questions/68795453/git-sh-exe-docker-run-no-such-file-or-directory
> 
> So I'm not sure if the single quotes work as well, it would have to be tested.

I edited the Makefile and replaced  /bin/sh with  //bin/sh (double slashes). 
I run the test: `make refresh_product_tags` 
And it works fine.
![make refresh_products_tag Success](https://user-images.githubusercontent.com/65444978/164510712-0561d0e9-4c18-43ff-b71d-36275ef7262a.jpg)

Reply:
Yes, and I tested, it is ok on linux.

So now you can remove this trouble shouting entry @pradumna-gautam, and it will be allright.